apps/sms/js/activity_handler.js

```diff
@@ -294,70 +293,56 @@ var ActivityHandler = {
       };
 
     if (Compose.isEmpty()) {
-      this.launchComposer(activity);
-    } else {
-      // ask user how should we do
-      ActivityHandler.displayUnsentConfirmation(activity);
+      return this.launchComposer(activity);
```

Comment:
yeah it's a case by case choice; in big functions it's not good, but big functions should not happen either. :)